auto_submit/lib/model/appengine/commit.dart

```diff
@@ -0,0 +1,120 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
rm? Seems like you pulled all the models in which is what we want to avoid. Please only copy the minimum necessary

Reply:
rm